zetaclient/testutils/testdata.go

```diff
@@ -38,16 +37,19 @@ func SaveObjectToJSONFile(obj interface{}, filename string) error {
 }
 
 // LoadObjectFromJSONFile loads an object from a file in JSON format
-func LoadObjectFromJSONFile(obj interface{}, filename string) error {
+func LoadObjectFromJSONFile(obj interface{}, filename string) {
 	file, err := os.Open(filepath.Clean(filename))
 	if err != nil {
-		return err
+		panic(err)
```

Comment:
Same, would be better to not use panic. even for testing

Reply:
https://github.com/zeta-chain/node/pull/1942/commits/edc9be607f5f551c3756277625beed1cf481c410